compiler/qsc_fir/src/fir.rs

```diff
@@ -765,6 +759,56 @@ impl Display for CallableDecl {
         write!(indent, "\ninput: {}", self.input)?;
         write!(indent, "\noutput: {}", self.output)?;
         write!(indent, "\nfunctors: {}", self.functors)?;
+        write!(indent, "\nimplementation: {}", self.implementation)?;
+        Ok(())
+    }
+}
+
+/// A callable implementations.
+#[derive(Clone, Debug, PartialEq)]
+pub enum CallableImplementation {
```

Comment:
nitpick: the name is a bit long, so following the patterns of abbreviations we used elsewhere this could be `CallableImpl` instead.

Reply:
Renamed.